lib/evmone_precompiles/bls.cpp

```diff
@@ -47,13 +78,31 @@ void store_result(uint8_t _rx[64], uint8_t _ry[64], const blst_p1* out) noexcept
     std::memset(_ry, 0, FP_BYTES_OFFSET);
     blst_bendian_from_fp(&_ry[FP_BYTES_OFFSET], &result.y);
 }
+
+/// Stores p2 point in two 128-bytes arrays with big endian encoding zero padded.
+void store_p2(uint8_t _rx[128], uint8_t _ry[128], const blst_p2* out) noexcept
+{
+    blst_p2_affine result;
+    blst_p2_to_affine(&result, out);
+
+    std::memset(_rx, 0, FP_BYTES_OFFSET);
```

Comment:
This is kinda 2x `store_p1`. So if you separate "to affine" from store you could reuse `store_p1`.